src/phrases/phrases-nn.js

```diff
@@ -952,16 +953,16 @@ const phrases = {
   },
   validation: {
     isRequired: '{label} er påkrevd.',
-    bothFields: 'En {labelLowerCase} må inneholde alle felter.',
+    bothFields: 'Ein {labelLowerCase} må inneholde alle felt.',
     isNumeric: '{label} må inneholde tall.',
-    maxLength: '{label} kan ikkje ha meir enn {maxLength, number} tegn.',
-    minLength: '{label} må ha minst {minLength, number} tegn.',
+    maxLength: '{label} kan ikkje ha meir enn {maxLength, number} teikn.',
+    minLength: '{label} må ha minst {minLength, number} teikn.',
     url: '{label} må inneholde ei gyldig lenke.',
     urlOrNumber: '{label} må inneholde ei gyldig lenke eller artikkel-id.',
     dateBeforeInvalid: '{label} kan ikkje være etter {afterLabel}.',
     dateAfterInvalid: '{label} kan ikkje være før {beforeLabel}.',
     minItems:
-      '{label} feltet må minst inneholde {minItems, plural, one{en} other{# ulike}} {labelLowerCase}.',
+      '{label} feltet må minst inneholde {minItems, plural, one{ein/eit} other{# ulike}} {labelLowerCase}.',
```

Comment:
Skulle egentlig vore eitt
```suggestion
      '{label} feltet må minst inneholde {minItems, plural, one{ein/eitt} other{# ulike}} {labelLowerCase}.',
```